webapp/packages/plugin-data-viewer-result-set-grouping/src/Actions/ACTION_DATA_VIEWER_GROUPING_CONFIGURE.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { createAction } from '@cloudbeaver/core-view';
+
+export const ACTION_DATA_VIEWER_GROUPING_CONFIGURE = createAction('data-viewer-grouping-configure', {
+  label: 'plugin-data-viewer-result-set-grouping_action_configure',
+  tooltip: 'plugin-data-viewer-result-set-grouping_action_configure_tooltip',
```

Comment:
please rename localization keys `plugin-data-viewer-result-set-grouping_action_configure` use `_` instead of `-`